EXPO/screens/HomeScreen.js

```diff
@@ -1,32 +1,35 @@
-import React from 'react';
+import React from "react";
 import {
   Image,
   Platform,
   ScrollView,
   StyleSheet,
   Text,
   TouchableOpacity,
-  View,
-} from 'react-native';
-import { WebBrowser } from 'expo';
+  View
+} from "react-native";
+import { WebBrowser } from "expo";
 
-import { MonoText } from '../components/StyledText';
+import { MonoText } from "../components/StyledText";
+import NetworkManager from "../src/network/network-manager";
 
 export default class HomeScreen extends React.Component {
   static navigationOptions = {
-    header: null,
+    header: null
   };
-
   render() {
     return (
       <View style={styles.container}>
-        <ScrollView style={styles.container} contentContainerStyle={styles.contentContainer}>
+        <ScrollView
+          style={styles.container}
+          contentContainerStyle={styles.contentContainer}
+        >
           <View style={styles.welcomeContainer}>
             <Image
               source={
                 __DEV__
-                  ? require('../assets/images/robot-dev.png')
-                  : require('../assets/images/robot-prod.png')
+                  ? require("../assets/images/robot-dev.png")
```

Comment:
extract this outside component as this will hit performance of component on update